lib/health_check/s3_health_check.rb

```diff
@@ -27,19 +27,14 @@ def check
 
       private
 
+      # We already assume you are using Rails.  Let's also assume you have an initializer
+      # created for your Aws config.  We will set the region here so you can use an
+      # instance profile and simply set the region in your environment.
       def configure_client
-        return unless defined?(Rails)
+        ::Aws.config[:s3] = { force_path_style: true }
+        ::Aws.config[:region] ||= ENV['AWS_REGION'] || ENV['DEFAULT_AWS_REGION']
```

Comment:
Does somebody know if this modifies the global configuration and not just the configuration used by `health_check` gem?